### DISCUSSION TITLE
More customizable local playback

### DISCUSSION THREAD

**Discussion by sevmonster** [jcorporation/myMPD#825] [Ideas]
Currently Local Playback feature expects MPD HTTP output on same host as mpd for streaming. But if using other output like Icecast (I personally use Pipewire + some preprocessing around JACK + Liquidsoap) it is unwieldly to configure. I would like simple HTTPS URL alternative to MPD stream port. My audio endpoint is HTTPS and already reverse proxied through nginx, and hosted on another subdomain, so it is not conducive to proxy again.

**Comment by jcorporation** [comment]
This could be implemented with the new per partition streaming function for v10.

#826
